README.md

```diff
@@ -15,6 +15,10 @@ For a quick start just
 [download the pre-built package](https://www.erlang-solutions.com/downloads/download-mongooseim)
 that suits your platform.
 
+Download
+--------
+Feel free to download MongooseIM at: https://www.erlang-solutions.com/downloads/download-mongooseim
```

Comment:
Ah, misleading then... a "Quickstart guide" should be a guide, right? ;-) So, as a user, I would expect a doc...